pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
```

Comment:
to jest baza w pamieci, mozna tez do pliku wrzucic. natomiast jest latwiej, bo nie musze sobie sciagac mysql.
oczywiscie to jest tylko do testow bazka